src/main/java/io/github/moulberry/notenoughupdates/commands/dev/DevTestCommand.java

```diff
@@ -122,6 +126,32 @@ public void printStackTrace() {
 			Utils.addChatMessage(EnumChatFormatting.RED + DEV_FAIL_STRINGS[devFailIndex++]);
 			return;
 		}
+		if (args.length == 1 && args[0].equalsIgnoreCase("testprofile")) {
+			NotEnoughUpdates.INSTANCE.manager.apiUtils.newHypixelApiRequest("skyblock/profiles")
```

Comment:
We already have a method for this in APIUtils. No code duplication, please.

Reply:
i think we can keep this as it's just a dev test